.colab/00_lineapy_quickstart.ipynb

```diff
@@ -34,25 +34,27 @@
         },
```

Comment:
The intention behind `.colab/` folder is to "host" all Colab-executable notebooks here. In other words, our Colab integration is set up such that any notebook file put into `.colab/` is hosted on Colab. Our GH workflow copies select notebooks in `examples/` and puts them into `.colab/` such that they can be executed by the user.

Reply:
Since our GH workflow is copying the notebooks into the `.colab` folder, can it be put into `.gitignore`? Or would that mess up the integration?

Reply:
Not 100% sure, but the GH action is here: https://github.com/LineaLabs/lineapy/blob/5f29005daf698f2581851fab88466923b85b03f4/.github/workflows/sync_notebooks.yml#L35 

And it simply runs a `git add .` command so I suspect putting the folder in the `.gitignore` will break the action since the git add command won't pick up the `.colab` folder

Reply:
One alternative could be to delete `.colab` from the source code, update the action to `mkdir `.colab`;cp <relevant directory> .colab` and then the same action as mentioned. This would avoid the duplicate files in the source `.colab` folder. However, will leave it up to you if it feels too messy and confusing/not worth it.

Reply:
Ok I updated the action and the `.gitignore`. The only additional issue with this method is that I end picking up on `.colab/data/iris.csv` but that should be ok.

Reply:
@aayan636 Let's sync on this comment.

Reply:
Currently decided to leave the PR as it is, `.colab` folder gets updated by the GitHub action. We can review this behavior of repeated folder contents later.